pages/common/exrex.md

```diff
@@ -0,0 +1,37 @@
+# exrex
+
+> Generate all/random matching strings for a regular expression.
+> It can also simplify regular expressions.
+> More information: <https://github.com/asciimoo/exrex>.
+
+- Generate all possible strings that match a regular expression:
+
+`exrex {{'regular_expression'}}`
```

Comment:
We use the quotes outside the tokens #4363
```suggestion
`exrex '{{regular_expression}}'`
```

Reply:
Ha! I didn't know it was official.  I intentionally didn't do it in this one though because at the very last example, the brackets look mixed up in the regex...but I guess clients are supposed to remove them anyway, so I'll do this.